packages/node/src/lock/types.ts

```diff
@@ -0,0 +1,7 @@
+export interface ILockInterface {
```

Comment:
As suggested by @kt, however I added `lockName`.

Reply:
Ah yes, `lockName` makes good sense here. Originally I had conceptualized the interface `ILock`, which internally would supply its own lock key. In `ILockService` `lockName` would be provided as you suggest 👍